src/plugins/auto_batch/src/plugin.cpp

```diff
@@ -19,344 +15,360 @@
 #include "transformations/common_optimizations/dimension_tracking.hpp"
 #include "transformations/init_node_info.hpp"
 #include "transformations/utils/utils.hpp"
+OPENVINO_SUPPRESS_DEPRECATED_START
+#include "ie_ngraph_utils.hpp"
```

Comment:
remove the test with ov::auto_batch_timeout(-1), cause the type of this variable is unsigned int. The original code can work is because the -1 will be converted to 4294967295, and 4294967295 exceed the limitation of std::stoi() which throw exception. The behavior is not expected.

Reply:
@ilyachur  This header file is needed by 
`          OPENVINO_SUPPRESS_DEPRECATED_START
            for (auto&& input : reshaped->inputs()) {
                auto& rt_info = input.get_rt_info();
                auto it = rt_info.find("ie_legacy_td");
                if (it != rt_info.end()) {
                    auto td = it->second.as<InferenceEngine::TensorDesc>();
                    rt_info["ie_legacy_td"] =
                        InferenceEngine::TensorDesc(td.getPrecision(), input.get_shape(), td.getLayout());
                }
            }
            for (auto&& result : reshaped->get_results()) {
                auto output = result->input_value(0);
                auto& rt_info = output.get_rt_info();
                auto it = rt_info.find("ie_legacy_td");
                if (it != rt_info.end()) {
                    auto td = it->second.as<InferenceEngine::TensorDesc>();
                    rt_info["ie_legacy_td"] =
                        InferenceEngine::TensorDesc(td.getPrecision(), output.get_shape(), td.getLayout());
                }
            }
            OPENVINO_SUPPRESS_DEPRECATED_END`
The convert between 1.0 and 2.0 lost the rt info.

Reply:
Why do you need this header? This code doesn't contain conversion. You can include public legacy API instead of this header.

Reply:
Yes, will use `ie_layouts.h`.